packages/web/src/components/track/desktop/ConnectedTrackTile.module.css

```diff
@@ -57,6 +57,7 @@
   flex: 1 1 100%;
   min-width: 0;
   justify-content: flex-start;
+  margin-right: var(--unit-12);
```

Comment:
This is actually not the first instance of this pattern in the tile component. I think if you really wanted to be sure, you could link this value to the width of the "right side" absolutely-positioned content to make sure that if we change one, the other will match? Either way, not _that_ hacky.

Reply:
Probably deserves a comment, but honestly I think we revamp the whole tile later because it's gotten gnarly